src/v2/Apps/ViewingRoom/Components/ViewingRoomsFeaturedRail.tsx

```diff
@@ -16,19 +18,48 @@ export const ViewingRoomsFeaturedRail: React.FC<ViewingRoomsFeaturedRailProps> =
     })
     .filter(Boolean)
 
-  return featuredViewingRoomsForRail.length > 0 && (
-    <Box>
-      <Sans size="5">Featured</Sans>
+  const carouselHeight = 380
+  const carouselItemRender = (
+    {
+      heroImageURL,
+      slug,
+      title,
+      partner,
+      status,
+      distanceToOpen,
+      distanceToClose,
+    },
+    slideIndex
+  ) => {
+    const tag = getTagProps(status, distanceToOpen, distanceToClose)
+    return (
+      <Flex flexDirection="row">
+        {slideIndex !== 0 && <Spacer ml="15px" />}
+        <Link href={`/viewing-room/${slug}`} key={slug} noUnderline>
+          <MediumCard
+            image={heroImageURL}
+            title={title}
+            subtitle={partner.name}
+            tag={tag}
+          />
+        </Link>
+      </Flex>
+    )
+  }
+
+  return (
+    featuredViewingRoomsForRail.length > 0 && (
       <Box>
-        {featuredViewingRoomsForRail.map(vr => {
-          return (
-            <Sans size="3t" key={vr.slug}>
-              {vr.title}
-            </Sans>
-          )
-        })}
+        <Sans size="5">Featured</Sans>
+        <ViewingRoomCarousel
+          height={carouselHeight}
+          items={featuredViewingRoomsForRail}
+          itemRender={carouselItemRender}
```

Comment:
It might be better if these prop names (`items`, `itemRender`) were consistent with the underlying carousel props that they get passed to (`data`, `render`), versus adding specificity and introducing slight divergence.